gyp/pylib/gyp/xcode_emulation.py

```diff
@@ -1541,6 +1541,13 @@ def CLTVersion():
         except GypError:
             continue
 
+    regex = re.compile('Command Line Tools for Xcode\s+(?P<version>\S+)')
+    try:
+        output = GetStdout(['/usr/sbin/softwareupdate', '--history'])
```

Comment:
When tested on Travis CI, this will pass on `osx_image: xcode11.4` and `osx_image: xcode11.5` but will fail on all previous osx images listed at https://docs.travis-ci.com/user/reference/osx/#macos-version

Reply:
Fixed. The `except` block will now catch `OSError` if the command was not found and return `None`.